substrate/frame/system/src/lib.rs

```diff
@@ -244,6 +244,51 @@ pub mod pallet {
 			type BlockHashCount = frame_support::traits::ConstU64<10>;
 			type OnSetCode = ();
 		}
+
+		/// Default configurations of this pallet in a solo-chain environment.
+		///
+		/// ## Considerations:
+		///
+		/// By default, this type makes the following choices:
+		///
+		/// * Use a normal 32 byte account id, with a [`DefaultConfig::Lookup`] that implies no
+		///   'account-indexing' pallet is being used.
+		/// * Given that we don't know anything about the existence of a currency system in scope,
+		///   an [`DefaultConfig::AccountData`] is chosen that has no addition data. Overwrite this
+		///   if you use `pallet-balances` or similar.
+		/// * Make sure to overwrite [`DefaultConfig::Version`].
+		/// * 2s block time, and a default 5mb block size is used.
+		pub struct SolochainDefaultConfig;
+
+		#[frame_support::register_default_impl(SolochainDefaultConfig)]
```

Comment:
Is this our first production default-config?  
Maybe marking it experimental first?

Reply:
good idea, done.